compiler/one-cmds/one-init

```diff
@@ -71,14 +71,51 @@ def _get_backends_list():
 
     return backends_list
 
-
+# TODO Add support for TF graphdef and bcq
 def _get_parser(backends_list):
+    init_usage = ('one-init [-h] [-v] [-V] '
+                  '[-i INPUT_PATH] '
+                  '[-o OUTPUT_PATH] '
+                  '[-m MODEL_TYPE] '
+                  '[-b BACKEND] '
+                  # args for onnx model
+                  '[--input_img_layout LAYOUT] '
+                  '[--output_img_layout LAYOUT] '
+                  # args for backend driver
+                  '[--] [COMMANDS FOR BACKEND DRIVER]')
 
-    init_usage = 'NYI'
     parser = argparse.ArgumentParser(
-        description='Command line tool to generate initial cfg file.', usage=init_usage)
-
-    # TODO Add args into parser
+        description='Command line tool to generate initial cfg file. '
+        'Currently tflite and onnx models are supported.',
+        usage=init_usage)
+
+    _utils._add_default_arg(parser, include_config_file=False)
+
+    parser.add_argument(
+        '-i', '--input_path', type=str, help='full filepath of the input model file')
+    parser.add_argument(
+        '-o', '--output_path', type=str, help='full filepath of the output cfg file')
+    parser.add_argument(
+        '-m',
+        '--model_type',
+        type=str,
+        help=('type of input model: "onnx", "tflite". '
+              'If the file extension passed to --input_path is '
+              '".tflite" or ".onnx", this arg can be omitted.'))
```

Comment:
Q) this seems optional and if model is tflite or onnx and this can be omitted, why need this option?

Reply:
This option is from this comment: https://github.com/Samsung/ONE/pull/9259/files#r897520623